pkg/metrics/producers/scheduledcapacity/schedules.go

```diff
@@ -0,0 +1,106 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scheduledcapacity
+
+import (
+	"fmt"
+	"github.com/awslabs/karpenter/pkg/apis/autoscaling/v1alpha1"
+	"github.com/robfig/cron/v3"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// parseScheduleSpecIntoCrontab returns a crontab equivalent to the strongly-typed schedule
+func parseScheduleSpecIntoCrontab(pattern *v1alpha1.Pattern) string {
+	var minutes, hours, days, months, weekdays []string
+
+	if pattern.Minutes != nil {
+		minutes = strings.Split(*pattern.Minutes, ",")
+		for i, val := range minutes {
+			minutes[i] = strings.Trim(val, " ")
+		}
+	} else {
+		// Unspecified defaults to the 0 value
```

Comment:
This comment is repeated many times. I'd either put it at the top as a function comment or omit it entirely.

Reply:
Good point.